criu/sockets.c

```diff
@@ -605,14 +605,18 @@ int restore_socket_opts(int sk, SkOptsEntry *soe)
 	tv.tv_usec = soe->so_rcv_tmo_usec;
 	ret |= restore_opt(sk, SOL_SOCKET, SO_RCVTIMEO, &tv);
 
-	ret |= restore_bound_dev(sk, soe);
 	ret |= restore_socket_filter(sk, soe);
 
 	/* The restore of SO_REUSEADDR depends on type of socket */
 
 	return ret;
 }
 
+int restore_bound_opts(int sk, SkOptsEntry *soe)
```

Comment:
Why do we need to introduce a new function here? Maybe it is better to call restore_bound_dev directly?

Reply:
restore_bound_dev is a static method, can't be called directly for sk-inet.c